src/modules/drawing/lib/drawingUtils.js

```diff
@@ -104,6 +104,14 @@ export function formatAngle(value, digits = 2) {
 
 export function extractOpenLayersFeatureCoordinates(feature) {
     let coordinates = feature.getGeometry().getCoordinates()
```

Comment:
Surprisingly, this seems to work. Still, I would rather try to find another solution as this may have lots of undesired side effects. This seems to store all computed geodesic coordinates directly in the editable feature. This means that the kml will store lots useless coordinates. This may in turn have unintended effects on other code that tries to access these coordinates and expects the "correct" coordinates or on other code that accesses this helper function.
I would rather try to access the geodesic coordinates directly when doing the request to service alti (e.g. in featureprofile.vue).

Reply:
Good point.

Reply:
I checked, effectively this solution adds the geodesic to the kml drawing which is not really what we want. I'll add a `geodesicCoordinates` property to the `EditableFeature` object instead.